src/client/components/WebSockets/WebSockets.tsx

```diff
@@ -94,6 +102,7 @@ export const WebSocketProvider: React.FC = ({ children }) => {
         newSocket.on('startGame', () => {
             dispatch(clearChat());
             dispatch(push('/ingame'));
+            dispatch(startGameAction());
```

Comment:
This redux action (start game) is responsible for resetting the 'last played card' in the store